presto-pinot-toolkit/src/main/java/com/facebook/presto/pinot/PinotSplitManager.java

```diff
@@ -183,7 +160,7 @@ public ConnectorSplitSource getSplits(
     {
         PinotTableLayoutHandle pinotLayoutHandle = (PinotTableLayoutHandle) layout;
         PinotTableHandle pinotTableHandle = pinotLayoutHandle.getTable();
-        Supplier<PrestoException> errorSupplier = () -> new QueryNotAdequatelyPushedDownException(QueryNotAdequatelyPushedDownErrorCode.PQL_NOT_PRESENT, pinotTableHandle, connectorId);
+        Supplier<PrestoException> errorSupplier = () -> new QueryNotAdequatelyPushedDownException(PinotErrorCode.PINOT_PUSH_DOWN_QUERY_NOT_PRESENT, pinotTableHandle, connectorId);
```

Comment:
Is `QueryNotAdequatelyPushedDownException` created any where else ? Should we bury the error code `PinotErrorCode.PINOT_PUSH_DOWN_QUERY_NOT_PRESENT` inside the constructor then ?

Reply:
My feeling  is that this error code is somehow very high level. We should have more detailed errors in the future. So prefer having an error code here.